lcm/CMakeLists.txt

```diff
@@ -22,9 +22,10 @@ set(lcm_install_headers
   eventlog.h
   lcm.h
   lcm_coretypes.h
+  lcm_version.h
```

Comment:
@mwoehlke-kitware I've just added another commit to also install lcm_version.h - otherwise downstream projects that use LCM as system-installed fail to use created lcmtype headers